modules/nf-core/pharokka/installdatabases/tests/main.nf.test

```diff
@@ -0,0 +1,22 @@
+nextflow_process {
+
+    name "Test Process PHAROKKA_INSTALLDATABASES"
+    script "../main.nf"
+    process "PHAROKKA_INSTALLDATABASES"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pharokka"
+    tag "pharokka/installdatabases"
+
+    test("No input required") {
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
```

Comment:
I don't know how frequently the databases get updated. I think snapshotting databases will cause failed tests in the future without a clear reason as they just got updated.

Reply:
Keep an eye on the [discussion](https://nfcore.slack.com/archives/C049MBCEW06/p1702888888959199). It's suggested to use regex for date files and see if they exist.

I think it's best to list the files and check for existence.

Reply:
I like your suggestion and did some experimenting around with it. However, because the output is a directory (`pharokka_db/`), which is required for input to the pharokka module, the assertions to determine whether files exist has been returning `java.lang.NullPointerException` because the `process.out.pharokka_db` only contains the directory path and not file paths. I am sure we could figure out a way around this, but since this tools databses are updated infrequently, and are actually hardcoded into each version release (there is no searching for the latest database, only an explicit path to a Zenodo database) I think the snapshot is okay for now. Does that work for you?

Reply:
Yeah makes sense! Updating the database snapshot is not a big issue is if it's linked to the tools version as the tool version itself is snapshotted.  

(fyi there already a couple of threads in nf-test slack channel that discuss how to check for nested files of an output dir, in case you would wan to use it in the future)

Reply:
I'll definitely look into those discussions. That could be very helpful in the future!